hera_mc/geo_handling.py

```diff
@@ -286,18 +272,32 @@ def print_loc_info(self, loc_list, verbosity='h'):
             print("No locations found.")
             return False
         for a in loc_list:
-            if verbosity == 'm' or verbosity == 'h':
-                print('station_name: ', a.station_name)
-                print('\teasting: ', a.easting)
-                print('\tnorthing: ', a.northing)
-                print('\tlon/lat:  ', a.lon, a.lat)
-                print('\televation: ', a.elevation)
-                print('\tstation description ({}):  {}'.format(a.station_type, a.desc))
-                print('\tcreated:  ', cm_utils.get_time_for_display(a.created_date))
-            elif verbosity == 'l':
-                print(a)
+            print('station_name: ', a.station_name)
+            print('\teasting: ', a.easting)
+            print('\tnorthing: ', a.northing)
+            print('\tlon/lat:  ', a.lon, a.lat)
+            print('\televation: ', a.elevation)
+            print('\tstation description ({}):  {}'.format(a.station_type_name, a.desc))
+            print('\tcreated:  ', cm_utils.get_time_for_display(a.created_date))
         return True
 
+    def parse_station_types_to_check(self, sttc, add_stations):
+        self.get_station_types(add_stations=add_stations)
+        if isinstance(sttc, (str, unicode)):
+            if sttc.lower() == 'all':
+                return self.station_types.keys()
+            else:
+                sttc = [sttc]
+        sttypes = []
```

Comment:
It might be better to make this a `set` rather than a list, in case the passed-in specification works out to include redundant station types.